docs/semgrep-ci/network-broker.md

```diff
@@ -0,0 +1,134 @@
+---
+slug: network-broker
+title: Semgrep Network Broker
+hide_title: false
+description: Learn how to set up the Semgrep Network Broker, which facilitates secure access between Semgrep and your private network.
+tags:
+  - Semgrep in CI
+  - Network broker
+---
+
+# Set up the Semgrep Network Broker
+
+The Semgrep Network Broker facilitates secure access between Semgrep and your private network. It accomplishes this by establishing a WireGuard VPN tunnel with the Semgrep infrastructure, then proxying **inbound** HTTP requests from Semgrep to your network through this tunnel. This approach allows Semgrep to interact with on-premise resources without exposing them to the public internet.
+
+Examples of inbound traffic include:
+
+- Pull request (PR) or merge request (MR) comments
+- Jira integrations
+- Webhooks
+
+:::info Subscriber availability
+The Semgrep Network Broker is available to Enterprise tier users.
+:::
+
+## Prerequisites
```

Comment:
```suggestion
## Prerequisites and feature availability
```

Maybe also add a very short tidbit on what SCMs this feature is available for use? 
"This feature is available to GitHub and GitLab..."